File: src/test/java/me/rasztabiga/fridgy/products/service/mapper/GroceryItemMapperTest.java
package me.rasztabiga.fridgy.products.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class GroceryItemMapperTest {

    private GroceryItemMapper groceryItemMapper;

    @BeforeEach
    public void setUp() {
        groceryItemMapper = new GroceryItemMapperImpl();
    }
}

File: src/test/kotlin/me/rasztabiga/fridgy/products/service/mapper/GroceryItemMapperTest.kt
package me.rasztabiga.fridgy.products.service.mapper

import org.junit.jupiter.api.BeforeEach

class GroceryItemMapperTest {

    private lateinit var groceryItemMapper: GroceryItemMapper

    @BeforeEach
    fun setUp() {
        groceryItemMapper = GroceryItemMapperImpl()
    }
}
